README.md

```diff
@@ -114,6 +114,9 @@ $token = $session->generateToken(array(
 
 ## Working with Archives
 
+**Important:** You can only archive sessions that use the OpenTok Media Router
```

Comment:
interesting, are we receiving feedback that this is not clear? i feel that between the reference documentation, archiving sample, and error messaging we've pretty much got it covered. i want to avoid turning the readme into a comprehensive reference, and if anything i'm hoping to reduce the length of the prose discussions in order to have the "happy path" and most common usages stand out.